content/alpha_edition/io_playability.md

```diff
@@ -2,7 +2,7 @@
 
 ## Pinout and Hardware Diagram
 ### Front
-<center>![Pinout Top](/assets/images/alphadelta top pinout 20200811.png)</center>  
+<center>![Pinout Top](/assets/images/alpha delta top pinout 20200811.png)</center>  
```

Comment:
This should be reverted, because that image is called "alphadelta top pinout 20200811.png".
Otherwise you should change the name of the image, and also the name and link of "alphadelta bot pinout 20200811.png"